### DISCUSSION TITLE
Add Purchase price

### DISCUSSION THREAD

**Discussion by tnemrap** [inventree/InvenTree#6869] [Q&A] [answered]
I added some parts to stock by add stock item.
I forget to add purchase price.
What is the way to add Purchace price to a part?

**Comment by SchrodingersGat** [comment, marked as answer]
You can edit the stock item manually to add in purchase price information

![image](https://github.com/inventree/InvenTree/assets/10080325/9e6face7-e346-42e7-a56a-67a91be3f9d2)
